sensors/jira_sensor.yaml

```diff
@@ -12,15 +12,15 @@
         properties:
           project:
             type: "string"
-          issue_name:
-            type: "string"
-          issue_url:
+          id:
             type: "string"
-          created:
+          expand:
             type: "string"
-          assignee:
-            type: "string"
-          fix_versions:
+          issue_name:
```

Comment:
I would be careful about removing existing keys from the schema - they presumably worked for somebody, so we should keep them unless they cause problems.

Additionally, since you mention that the `assignee` field in the payload is an object, we should model that key as well.

Not that none of these properties are required since we do not have a `required: []` key anywhere in the schema, so modeling a key that may not appear should not cause issues (famous last words...).